### PR TITLE
Add Oracle Cloud event handlers for targeted refresh

### PR DESCRIPTION
**Pull request by agrare**
Required for: https://github.com/ManageIQ/manageiq-providers-oracle_cloud/pull/14

TODO:
- [x] Fill out event types that we want to trigger a targeted refresh

### CODE REVIEW

File: content/automate/ManageIQ/System/Event/EmsEvent/ORACLE.class/__class__.yaml [hunk 1]
```diff
@@ -0,0 +1,433 @@
+---
```
Comment (agrare) [inline content/automate/ManageIQ/System/Event/EmsEvent/ORACLE.class/__class__.yaml:1]: I have no idea what this file does :laughing: I copied it from another provider class

**Comment by lfu** [comment]
Please follow the steps in [Creating a Pull Request for enhancements to ManageIQ Automate Domain](http://talk.manageiq.org/t/creating-a-pull-request-for-enhancements-to-manageiq-automate-domain/1041).

**Comment by agrare** [comment]
Maybe a good idea to include that in the README in this repo?  I don't usually think to search talk.manageiq.org when making a pull-request

**Comment by agrare** [comment]
@lfu are you sure that is correct?  I followed those steps and re-exported the domain with `$ rake evm:automate:export DOMAIN=ManageIQ EXPORT_DIR=../manageiq-content/content/automate OVERWRITE=true` and I get _a lot_ of changes but none to the ORACLE.class

**Comment by agrare** [comment]
@lfu now more files are failing, I don't think this is a problem with hand editing

**Comment by lfu** [comment]
`rake evm:automate:export` should work. But I would prefer using the automate import/export function from UI to import the new class then export it. Then I would copy the exported new class to git repo to commit them.

**Comment by agrare** [comment]
@lfu the problem is my local version of YAML strips the trailing whitespace out so no matter what method I use it will fail on travis.  This doesn't mean they are hand edited it means the version of libyaml on travis is old

**Comment by agrare** [comment]
Okay got the travis ruby version fix in master, taking this out of WIP

**Review by lfu — APPROVED** [review]

**Comment by Fryguy** [comment]
Backported to `morphy` in commit 9e98e64d0f0297313686478fc63137e181b520ec.

```text
commit 9e98e64d0f0297313686478fc63137e181b520ec
Author: Lucy Fu <lufu@redhat.com>
Date:   Sat Mar 20 16:35:52 2021 -0400

    Merge pull request #685 from agrare/add_oracle_cloud_event_handlers
    
    Add Oracle Cloud event handlers for targeted refresh
    
    (cherry picked from commit c0de5acdc1f2ed164d03232c3c6d80ddc0c3b14e)
```

**Pull request merged**